### PR TITLE
fix morphologyIndex type in morphology release

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

**Comment by genric** [comment]
Then I would probably prefer to have MorphologyModelIndex entity rather than generic entity. I also don't mind simplifying it to just embedded distribution object and provenance data related to MorphologyRelease only.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "nodekind": "sh:BlankNodeOrIRI",
       "and": [
         {
-          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json]: Is a MEModelReleaseShape a Model ? Only models can extends the ModelInstanceShape.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
       "nodekind": "sh:BlankNodeOrIRI",
       "and": [
         {
-          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json]: Same comment as above.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json [hunk 3]
```diff
@@ -33,8 +33,7 @@
               "path": "nsg:morphologyIndex",
               "name": "Morphology release index file",
               "description": "Morphology release index file.",
-              "seeAlso": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape",
-              "class": "nsg:Entity"
+              "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json]: I would not recommend this approach. Instead, submit the morphologyIndex as an entity (so against the schema [neurosciencegraph/core/entity/v0.1.0.json](https://github.com/INCF/neuroshapes/blob/master/modules/core/src/main/resources/schemas/neurosciencegraph/core/entity/v0.1.0.json)) and then refer to it here as nsg:morphologyIndex value.

The shape could be:
```
{
              "path": "nsg:morphologyIndex",
              "name": "Morphology release index file",
              "description": "Morphology release index file.",
              "class":"nsg:Entity", (or nsg:MorphologyIndex is you want a specific type)
              "seeAlso": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
}
```

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:mediaType",
+              "name": "Media type",
+              "description": "",
+              "in": [
+                "application/mod"
```
Comment (MFSY) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1.json]: application/vdn.bbp.mod ?

Comment (jdcourcol) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1.json]: this is not a bbp thing here.
it is a Neuron simulator type of file

Comment (MFSY) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/synapserelease/v0.1.1.json]: OKay.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json (hunk 1 above)
Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json]: Yes, MEModelReleases need to be annotated with brain region and species information.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json (hunk 2 above)
Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json]: Same as above.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json (hunk 3 above)
Comment (genric) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json]: Done.

**Comment by genric** [comment]
We need brain region and species annotations for the release objects this is what this commit is suppose to provide. @jdcourcol , @apdavison could you please express your opinion if it makes sense for the EModelRelease, MEModelRelease, MorphologyRelease, SynapseRelease to be ModelInstance and hence have brainRegion, species properties. As an alternative, suggested by @MFSY, we could have a Release entity which would have region, species properties attached and then all the specific release entities would extend that Release entity. Thanks.

**Comment by genric** [comment]
Well I went for the second option, so now this commit includes ModelRelease which is a nsg:Entity, it has the brain region and species properties. All the specific release entities will extend ModelRelease.

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.1.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/simulation/modelrelease/v0.1.1",
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:EModelReleaseShape",
+      "@type": "sh:NodeShape",
+      "label": "EModel release shape. Output of BLuePyOpt",
```
Comment (jdcourcol) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.1.json]: typo BLuePyOpt -> BluePyOpt

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodel/v0.1.1.json [hunk 6]
```diff
@@ -42,6 +42,12 @@
               "path": "nsg:modelScript",
               "name": "EModel script",
               "class": "nsg:EModelScript"
+            },
+            {
+              "path": "dcterms:isPartOf",
+              "name": "Part of",
+              "description": "The memodel release this emodel is part of.",
```
Comment (jdcourcol) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodel/v0.1.1.json]: emodel => memodel

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json [hunk 7]
```diff
@@ -43,7 +44,8 @@
               "name": "Memodel index",
               "description": "Memodel release index file.",
               "editorialNote": "Check the files to attach to this entity .The following node constraint can be add later: {{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape.",
-              "class": "nsg:Entity"
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/modelindex/v0.1.1/shapes/ModelIndexShape",
+              "class": "nsg:ModelIndex"
```
Comment (jdcourcol) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json]: why ModelIndex ?
why not MEModelReleaseIndex ?

**Comment by genric** [comment]
Can we merge this?

**Pull request merged**